frontend/src/layers/ThQ.ts

```diff
@@ -52,19 +48,32 @@ export class ThQ {
 export const value = (boundaryLayerDepth: number, uWind: number, vWind: number, totalCloudCover: number): number => {
   // Boundary Layer Depth
   const bld = boundaryLayerDepth;
-  const blhCoeff = Math.min(bld / 800, 1); // >800 m = 100%
+  // coeff is 50% for a boundary layer depth of 400 m
+  const bldCoeff = logistic(bld, 400, 4);
```

Comment:
You can see what this function looks like here: https://www.wolframalpha.com/input?i=plot+1%2F%281+%2B+exp%28-%28x-u%29%2Fs%29%29%2C+x%3D0..1500%2C+u%3D400%2C+s%3D400%2F4